### DISCUSSION TITLE
Pure UEFI not working

### DISCUSSION THREAD

**Discussion by BujuHackTosh** [SkyrilHD/Dell-E7250-Hackintosh#34] [Q&A]
The pure UEFI has glitches / horizontal lines on the display. Is there any other alternative of booting pure UEFI?

**Comment by SkyrilHD** [comment]
Read this: https://github.com/SkyrilHD/Dell-E7250-Hackintosh#graphical-glitches-in-macos

**Reply by BujuHackTosh** [reply]
> Read this: https://github.com/SkyrilHD/Dell-E7250-Hackintosh#graphical-glitches-in-macos

Unfortunately, I have attempted to use this guide but I am still getting glitches as soon as the PC boots to the lockscreen. I already have used the One Key HiDpi but still no success.

**Reply by SkyrilHD** [reply]
HiDPI with EDID?

https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/discussions/6

**Reply by BujuHackTosh** [reply]
Yes, tried using that option (Patch) still no success. Weirdly, there are no glitches if the laptop is from sleep. The glitches are only present from a cold boot.

**Comment by BujuHackTosh** [comment]
Is there any updates?

**Reply by SkyrilHD** [reply]
Sadly, No. As I don't have this issue, I can't do any debugging. The "fix" provided were confirmed to be working, no idea why it is not working on yours.
